docker-compose.yml

```diff
@@ -16,6 +16,8 @@ services:
     volumes:
       - ./:/app
       - $HOME/.m2:/Users/1000/.m2 # Required for jparest dependencies (artifactory)
+      - keystore:/keystore
+      - ./repository:/Users/1000/.m2/repository
```

Comment:
This makes searches really awful because the repository folder created as a result isn't excluded be default by any IDE. 
This shouldn't be needed as the repository is usually under $Home/.m2. Since I believe this fixes a specific issue for someone could we not make it the default. We could map $REPOSITORY:/Users/1000/.m2/repository and default REPOSITORY=$Home/.m2/repository. That way if someone needs to override it they can just add a local env file that points REPOSITORY wherever they like

Reply:
Have opened a PR to remove this as discussed.
https://github.com/UKHomeOffice/callisto-timecard-restapi/pull/87